clients/avro/pom.xml

```diff
@@ -70,12 +70,13 @@ http://maven.apache.org/xsd/maven-4.0.0.xsd">
     </dependency>
     <dependency>
        <groupId>org.slf4j</groupId>
-       <artifactId>slf4j-log4j12</artifactId>
+       <artifactId>slf4j-reload4j</artifactId>
    </dependency>
    <!-- Use a repackaged version of log4j with security patches. Default log4j v1.2 is a transitive dependency of slf4j-log4j12, but it is excluded in common/pom.xml -->
     <dependency>
         <groupId>io.confluent</groupId>
         <artifactId>confluent-log4j</artifactId>
+        <version>1.2.17-cp10</version>
```

Comment:
you _should_ be able to remove the `confluent-log4j` dependency and use this instead:

```
    <dependency>
      <groupId>ch.qos.reload4j</groupId>
      <artifactId>reload4j</artifactId>
      <version>${reload4j.version}</version>
    </dependency>
```